x-pack/plugins/cloud_security_posture/public/pages/findings/findings.tsx

```diff
@@ -101,6 +109,26 @@ export const Findings = () => {
             </h1>
           </EuiTitle>
           <EuiSpacer />
+          {!userHasDismissedCallout && (
```

Comment:
what about the case when user either acted on the callout and installed the integration or have already had the integration installed? I think it would be better not to show this callout if wiz integration already installed (or we can be even more precise and check if there is latest wiz index)

Reply:
This process of logic has been decided to move to a later stage precisely because there are more considerations, like integration versions and such. This logic will also be used to determine if we want to display the vendor column in the table. I felt it was enough logic to push into a second-stage issue. Closing the box once is not so annoying that it’s worth delaying more important features. Considering all this automated decision logic is always a bit messy to write and test, especially when combined with local storage state and such.

Reply:
here is the issue btw: https://github.com/elastic/kibana/issues/187234